presentation/build.gradle

```diff
@@ -76,6 +76,13 @@ dependencies {
   androidTestCompile presentationTestDependencies.mockito
   androidTestCompile presentationTestDependencies.dexmaker
   androidTestCompile presentationTestDependencies.dexmakerMockito
-  androidTestCompile presentationTestDependencies.espresso
-  androidTestCompile presentationTestDependencies.testingSupportLib
+  androidTestCompile (presentationTestDependencies.espresso) {
+    exclude group: 'com.android.support', module: 'support-annotations'
```

Comment:
Because there is a conflict with dependencies, It's cause because there is different versions of android support annotation. If I remove this line a I get this error :

```Warning:Conflict with dependency 'com.android.support:support-annotations'. Resolved versions for app (21.0.3) and test app (23.0.1) differ.```